src/test/java/edu/wisc/my/messages/controller/MessagesControllerTest.java

```diff
@@ -28,35 +28,4 @@ public void siteIsUp() throws Exception {
       .andExpect(content().contentTypeCompatibleWith("application/json"))
       .andExpect(content().json("{\"status\":\"up\"}"));
   }
-
-  @Test
-  public void filteredMessagesIncludesAMessage() throws Exception {
```

Comment:
Hmm. Rather than removing this test, would it be better to update it to test for the new expected JSON format? Testing that we get the expected JSON back seems reassuring?